### ISSUE TITLE
sourcemap refence wrong with manual destination

### ISSUE THREAD

**Issue by coussej** [egoist/rollup-plugin-postcss#32]
When specifying a manual destination for the css output file, the reference to the sourcemap is wrong. 

my rollup.js config:

```javascript
export default {
  entry: 'src/main.js',
  dest: 'dist/main.js',
  format: 'iife',
  sourceMap: true,
  plugins: [
    ...
    postcss({
      sourceMap: true,
      extract : 'dist/bundle.css'
    }),
    ...
  ],
}

```
The file bundle.css.map is created, but bundle.css references to main.css.map:
```
/*# sourceMappingURL=main.css.map */
```

This is because of the following line of code (https://github.com/egoist/rollup-plugin-postcss/blob/master/src/index.js#L44):

```javascript
          css += `\n/*# sourceMappingURL=${fileName}.css.map */`
          promises.push(fs.writeFile(`${cssOutputDest}.map`, map))
```
${fileName}  is always taken from the automatic destination, while the ${cssOutputDest} is taken from manualDest when it is available. 

Suggested fix:
```javascript
          fileName = path.basename(cssOutputDest)
          css += `\n/*# sourceMappingURL=${fileName}.map */`
          promises.push(fs.writeFile(`${cssOutputDest}.map`, map))
```

**Comment by lmihaidaniel** [comment]
Thank you for catching this issue and providing a suggestion fix. We will tackle this down.

**Comment by coussej** [comment]
I created a pull request with the qucik fix, but feel free to ignore it if you want to tackle it in a different way :-)

**Comment by lmihaidaniel** [comment]
Fixed in #34
